src/lib/people/team/team.ts

```diff
@@ -0,0 +1,47 @@
+import * as v from 'valibot';
+
+export const committees = [
+    'Executive',
+    'Innovation',
+    'Service',
+    'External Relations',
+    'Membership & Internals',
+    'Branding & Creatives',
+    'Engineering',
+];
+
+const MemberSchema = v.object({
+    name: v.string(),
+    title: v.string(),
+    committee: v.pipe(
+        v.string(),
+        v.check(i => committees.includes(i), 'invalid committee'),
+    ),
+    socials: v.optional(v.record(v.string(), v.pipe(v.string(), v.url()))),
+});
+
+export interface Member extends v.InferOutput<typeof MemberSchema> {
+    img: string;
+}
```

Comment:
One cool thing about TypeScript is that `type` declarations and `const` declarations exist in separate namespaces. We can actually get away with something like this.
```ts
export const Member = v.object({ ... });
export type Member = v.InferInput<typeof Member>;
```

Reply:
Hmm, I'll take note of being able to use `Member` for defining the schema, but I think I'll prefer `interface`s for the `Member ` type declaration since it needs to include `img` as well. Extending an interface seems more appropriate than extending a type. (Also, I realize we could rename `img` to `src` at this point)

Reply:
Yup, I'm on board with that. The `type` was used as an example, but I believe `interface` declarations are also in a separate namespace from variable names. It should work in the same way (if I recall correctly).